public-ui/src/app/data-browser/views/concept-chart/concept-chart-react.component.tsx

```diff
@@ -0,0 +1,407 @@
+                    tabIndex={tabIndex} key={index}>
+                    <span>{g}</span>
+                    <TooltipReactComponent tooltipKey={g}
+                    label='EHR Tooltip Hover' searchTerm={searchTerm}
+                    action={'Concept graph ' + g + ' tooltip hover on concept ' + concept.conceptName}>
+                    </TooltipReactComponent>
+                    </div>
+                );
+          })
+        }
+            </div>
+            {loading ? <Spinner /> : null}
+            {displayGraphErrorMessage
+                    ? <div className='graph-error-message'>
+                        <ErrorMessageReactComponent dataType='chart' />
+                      </div>
+                    : [
+                    <React.Fragment key={concept.conceptId}>
+                    {(isAnalysisLoaded && selectedChartAnalysis && countAnalysis && countAnalysis.genderCountAnalysis) &&
+                    graphToShow === 'Sex Assigned at Birth' ?
+                    <div className='chart' key='biosex-chart'>
+                        <BioSexChartReactComponent
+                              domain='ehr' genderAnalysis={selectedChartAnalysis}
+                              genderCountAnalysis={countAnalysis.genderCountAnalysis} selectedResult=''/>
+                    </div> :
+                    graphToShow === 'Age' ?
+                    <div className='chart' key='age-chart'>
+                        <AgeChartReactComponent domain='ehr' ageAnalysis={selectedChartAnalysis}
+                        ageCountAnalysis={countAnalysis.ageCountAnalysis} selectedResult=''/>
+                    </div> :
+                    graphToShow === 'Values' ?
+                    <div className='chart' key='values-chart'>
+                    {unitNames.map((unit, index) => {
+                        return (
+                            <div key={index} className={selectedUnit === unit ? 'active btn btn-link unit-choice' : 'btn btn-link unit-choice'}
+                            onClick={() => this.showMeasurementGenderHistogram(unit)}>{unit}</div>
+                        );
+                    })
+                    }
+                    <div>
+                    {mixtureOfValues ?
+                    noUnitValueButtons.map((noUnit, index) => {
+                        return <div key={index} className={selectedMeasurementType === noUnit ? 'active btn btn-link measurement-filter-choice' : 'btn btn-link measurement-filter-choice'}
+                        style={styles.measurementFilterChoice}
+                        onClick={() => this.showSpecificMeasurementTypeValues(noUnit)}>{noUnit}</div>;
+                    }) : null
+                    }
+                    </div>
+                    <div className='chart-container'>
+                    <div className='ehr-m-chart-layout' style={styles.ehrMChartLayout}>
+                    {(genderResults && toDisplayMeasurementGenderAnalysis) ?
```

Comment:
Should use `&&` instead of `?` here and drop `: null` below

Reply:
Changed this. thanks